packages/mutation-testing-elements/src/components/mutation-test-report-totals.ts

```diff
@@ -0,0 +1,185 @@
+      fill: currentColor;
+    }
+
+    table th.vertical-middle, table td.vertical-middle {
+      vertical-align: middle;
+    }
+  `];
+
+  public render() {
+    if (this.model) {
+      return html`
+          <table class="table table-sm table-hover table-bordered table-no-top">
+            ${this.renderHead()}
+            ${this.renderTableBody(this.model)}
+          </table>
+      `;
+    } else {
+      return undefined;
+    }
+  }
+
+  private renderHead() {
+    return html`<thead>
+  <tr>
+    <th colspan="2" style="width: 217px">
+      <div><span>File / Directory</span></div>
+    </th>
+    <th colspan="2">
+      <div><span>Mutation score</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Killed</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Survived</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Timeout</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># No coverage</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Runtime errors</span></div>
+    </th>
+    <th class="rotate text-center" style="width: 50px">
+      <div><span># Compile errors</span></div>
+    </th>
+    <th class="rotate rotate-width-70 text-center" style="width: 70px">
+      <div><span>Total detected</span></div>
+    </th>
+    <th class="rotate rotate-width-70 text-center" style="width: 70px">
+      <div><span>Total undetected</span></div>
+    </th>
+    <th class="rotate rotate-width-70 text-center" style="width: 70px">
+      <div><span>Total mutants</span></div>
+    </th>
+  </tr>
+</thead>`;
+  }
+
+  private renderTableBody(model: FileResultModel | DirectoryResultModel) {
+    const renderChildren = () => {
+      if (!model.representsFile) {
+        return model.childResults.map(childResult => {
+          let fullName: string = childResult.name;
+          while (!childResult.representsFile && childResult.childResults.length === 1) {
+            childResult = childResult.childResults[0];
+            fullName = pathJoin(fullName, childResult.name);
+          }
+          return this.renderRow(fullName, childResult, true);
+        });
+      } else {
+        return undefined;
+      }
+    };
+    return html`
+    <tbody>
+      ${this.renderRow(model.name, model, false)}
+      ${renderChildren()}
+    </tbody>`;
+  }
+
+  private renderRow(name: string, row: ResultModel, shouldLink: boolean) {
+    const mutationScoreRounded = row.totals.mutationScore.toFixed(2);
+    const coloringClass = this.determineColoringClass(row.totals.mutationScore);
+    const progressBarStyle = `width: ${mutationScoreRounded}%`;
+    return html`
+    <tr title="${row.name}">
+      <td style="width: 17px;" class="icon no-border-right">${row.representsFile ? svg.file : svg.directory}</td>
```

Comment:
I find the look of this element to be a little weird (see image). The `<` looks like it's clickable, but it's not. And because it is a `<` it also looks like a back button, even though it is for going 'deeper' into the directory structure

![image](https://user-images.githubusercontent.com/10114577/54156778-39869980-4447-11e9-962e-ccc89f0ea734.png)

Reply:
I don't see the `<`:

![image](https://user-images.githubusercontent.com/1828233/54184706-8f8b2980-44a7-11e9-8e36-dae34d43fdeb.png)

Can you send me a deep link?

Reply:
Weird. I just ran the report with `npm start` and clicked through it. Maybe there's a difference between dev and prod?

Reply:
Ok I am not sure what I was doing yesterday, but I don't have the issue anymore

Reply:
I think you typed the `>` yourself 😅